.github/workflows/soak-testing.yml

```diff
@@ -0,0 +1,237 @@
+# Pre-requisites:
+# - AWS Account with correct permissions
+# - `gh-pages` branch
+# - AWS Account needs to add the LOG_GROUP_NAME defined below
+
+name: Soak Testing
+on:
+  workflow_dispatch:
+    inputs:
+      target_commit_sha:
+        description: 'The commit SHA on this repo to use for the Soak Tests.'
+        required: true
+      test_duration_minutes:
+        description: 'The duration of the Soak Tests in minutes.'
+        required: true
+        default: 300
+  schedule:
+    - cron: '0 15 * * *'
+env:
+  # NOTE: The configuration of `APP_PROCESS_EXECUTABLE_NAME` is repo dependent
+  APP_PROCESS_EXECUTABLE_NAME: EXAMPLE_OF_SOMETHING
+  AWS_DEFAULT_REGION: us-east-1
+  DEFAULT_TEST_DURATION_MINUTES: 300
+  HOSTMETRICS_INTERVAL_SECS: 600
+  CPU_LOAD_THRESHOLD: 50
```

Comment:
Oh don't review this PR yet! I have it in Draft mode because the PR you linked needs to be merged first 😃

You'll notice that the base of that PR _is_ this branch. So when that PR is merged this value will change.